packages/buttons/src/types/index.ts

```diff
@@ -62,6 +62,11 @@ export interface IAnchorProps
    * safe [cross-origin destination link](https://web.dev/external-anchors-use-rel-noopener/).
    **/
   isExternal?: boolean;
+  /**
+   * Allows a customized/translated text label to be passed to the external link icon,
+   * making that icon accessible to assistive technology
+   **/
+  externalIconTextLabel?: string;
```

Comment:
```suggestion
  externalIconLabel?: string;
```

Following other components (`<Avatar statusLabel="...">` for example), let's drop the redundant "Text" from the prop name.